query/src/main/scala/filodb/query/exec/RangeVectorTransformer.scala

```diff
@@ -240,4 +241,47 @@ final case class SortFunctionMapper(function: SortFunctionId) extends RangeVecto
       source
     }
   }
-}
\ No newline at end of file
+}
+
+final case class AbsentFunctionMapper(columnFilter: Seq[ColumnFilter], rangeParams: RangeParams, metricColumn: String)
+  extends RangeVectorTransformer {
+
+  protected[exec] def args: String =
+    s"columnFilter=$columnFilter rangeParams=$rangeParams metricColumn=$metricColumn"
+
+  def apply(source: Observable[RangeVector],
+            queryConfig: QueryConfig,
+            limit: Int,
+            sourceSchema: ResultSchema): Observable[RangeVector] = {
+
+      val resultRv = source.toListL.map { rvs =>
```

Comment:
You don't need `toListL` here, just use `countL` which saves creating a List.  Check the county

Reply:
Good point. Thanks!